tests/Tests/ORM/Functional/ReadOnlyTest.php

```diff
@@ -90,6 +90,26 @@ public function testReadOnlyQueryHint(): void
         self::assertTrue($this->_em->getUnitOfWork()->isReadOnly($user));
     }
 
+    public function testNotReadOnlyQueryHint(): void
+    {
+        $user = new ReadOnlyEntity('beberlei', 1234);
+
+        $this->_em->persist($user);
+
+        $this->_em->flush();
+        $this->_em->clear();
+
+        $dql = 'SELECT u FROM ' . ReadOnlyEntity::class . ' u WHERE u.id = ?1';
+
+        $query = $this->_em->createQuery($dql);
```

Comment:
```suggestion
        $query = $this->_em->createQuery('SELECT u FROM ' . ReadOnlyEntity::class . ' u WHERE u.id = ?1);
```

Reply:
Missing the end quote:
`$query = $this->_em->createQuery('SELECT u FROM ' . ReadOnlyEntity::class . ' u WHERE u.id = ?1');`

Reply:
Thanks, resolved!